src/main/java/org/folio/innreach/domain/service/impl/VirtualRecordServiceImpl.java

```diff
@@ -30,5 +34,7 @@ public void deleteVirtualRecords(UUID folioItemId, UUID folioHoldingId, UUID fol
     Optional.ofNullable(folioInstanceId).ifPresent(instanceService::delete);
     Optional.ofNullable(folioLoanId).ifPresent(loanService::delete);
     log.info("VirtualRecords Deleted");
+    log.info("deleteVirtualRecords execution ended at " + new Date());
```

Comment:
Probably should use the `{}` parameter rather than string concatenation on your log statements throughout.

Reply:
done